eox_core/api/v1/tests/integration/data/fake_users.py

```diff
@@ -1017,5 +1017,83 @@
+            "email": "privera29@protonmail.com",
+            "fullname": "Pedro Rivera",
+            "password": "pR7&2Ab#qZ@",
+            "activate_user": True,
+            "mailing_address": "305 Elm Street",
+            "year_of_birth": 1995,
+            "gender": "m",
+            "level_of_education": "p",
+            "city": "Miami",
+            "goals": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+        },
+        {
+            "username": "llopez17",
+            "email": "llopez17@protonmail.com",
+            "fullname": "Luisa Lopez",
+            "password": "lL3%8Mp@kF#",
+            "activate_user": True,
+            "mailing_address": "500 Oak Drive",
+            "year_of_birth": 1990,
+            "gender": "f",
+            "level_of_education": "p",
+            "city": "Los Angeles",
+            "goals": "Sed eget justo nec erat dignissim suscipit.",
+        },
+        {
+            "username": "jrodriguez42",
+            "email": "jrodriguez42@protonmail.com",
+            "fullname": "Juan Rodriguez",
+            "password": "jR9!5Eq*gT@",
+            "activate_user": True,
+            "mailing_address": "702 Maple Avenue",
+            "year_of_birth": 1983,
+            "gender": "m",
+            "level_of_education": "p",
+            "city": "New York",
+            "goals": "Vivamus ut sem consectetur, dignissim elit nec, placerat purus.",
+        },
+        {
+            "username": "smartinez23",
+            "email": "smartinez23@protonmail.com",
+            "fullname": "Sofia Martinez",
+            "password": "sM2@4Dl!hB#",
+            "activate_user": True,
+            "mailing_address": "810 Pine Road",
+            "year_of_birth": 1987,
+            "gender": "f",
+            "level_of_education": "p",
+            "city": "Chicago",
+            "goals": "Donec eget ante eu dolor sodales molestie.",
+        },
+        {
+            "username": "jmendoza35",
+            "email": "jmendoza35@protonmail.com",
+            "fullname": "Jorge Mendoza",
+            "password": "jM6@3Rw!nZ#",
+            "activate_user": True,
+            "mailing_address": "409 Cedar Street",
+            "year_of_birth": 1992,
+            "gender": "m",
+            "level_of_education": "p",
+            "city": "Houston",
+            "goals": "Pellentesque maximus ante nec sem malesuada, et interdum nisl iaculis.",
+        },
+        {
+            "username": "ggonzalez19",
+            "email": "ggonzalez19@protonmail.com",
+            "fullname": "Gabriela Gonzalez",
+            "password": "gG1@7Sw$pC#",
+            "activate_user": True,
+            "mailing_address": "201 Walnut Boulevard",
+            "year_of_birth": 1985,
+            "gender": "f",
+            "level_of_education": "p",
+            "city": "San Francisco",
+            "goals": "In ultrices sem nec turpis pretium, non consequat quam venenatis.",
+        },
```

Comment:
@mariajgrimaldi It seems to me that it is due to how the rest of the integration tests are implemented, since to use a fake user you use `self.user = next(FAKE_USER_DATA)`. Even the task in Jira talks about creating new users for each test.

Reply:
Done @magajh @mariajgrimaldi https://github.com/eduNEXT/edunext-internal-documentation/pull/230